clients/admin-ui/src/features/subject-request/ActivityTimeline.tsx

```diff
@@ -0,0 +1,73 @@
+import { Box, Flex, Text } from '@fidesui/react';
+import React from 'react';
+
+import { GreenCheckCircle } from '../common/Icon';
+import { PrivacyRequest } from '../privacy-requests/types';
+
+type ActivityTimelineProps = {
+  subjectRequest: PrivacyRequest;
+  setEventDetails: Function;
+};
+
+const ActivityTimeline = ({
```

Comment:
This component is mostly finished. It's waiting on some mockups to be finished. I talked with @seanpreston and we decided to leave it in the repo without being used because the work was already done here.